viewer/src/components/LdThumbnail.vue

```diff
@@ -66,6 +66,8 @@ export default class LdThumbnail extends Vue {
   width: $thumbnail-other-size;
   height: $thumbnail-other-size;
   text-align: center;
+  word-break: break-all;
```

Comment:
```suggestion
  word-break: break-word;
```

So that it tries to break long titles in a smart way first, breaking at characters only as a last resort.